### ISSUE TITLE
Request: update for older version of the games?

### ISSUE THREAD

**Issue by CronosD10** [SpencerMycek/KineticistExpandedElements#63]
Not sure if the "composite" blasts with negative/positive + any energy are working yet, but would it be possible to allow the new feature from updates to work with the 1.3.7 version? (honestly, just want to use this mod to the fullest with visual adjustments)

**Comment by SpencerMycek** [comment]
Just to understand the question: You're asking for a version of the mod built for 1.3.7?
Is that something you're still looking for?

**Comment by CronosD10** [comment]
Not exactly, the 1.3.7 i was refering to was the game version, it's one of the betas accesible on steam and it work with Visual Adjustments. Visual has been discontinued and all of my mods work on said version of the game, not sure how much coding would be needed to "downgrade" to older versions

**Comment by SpencerMycek** [comment]
Not entirely sure how much work it might be
But I can take a look, as this mod at one point was built for 1.3, since that's where P:WoTR was when I started. So I'll have to see what I can do
